src/ast.pyj

```diff
@@ -49,6 +49,7 @@ class Token(AST):
         'type': 'The type of the token',
         'subtype': 'The subtype of the token',
         'value': 'The value of the token',
+        '_value': 'The src value of the token',
```

Comment:
since RS replaces `or` with `||` we need to store original token value for cases when it is used as class method name